package.json

```diff
@@ -129,9 +129,11 @@
 		"@fluidframework/test-tools": "^0.2.3074",
 		"@microsoft/api-documenter": "^7.17.9",
 		"@microsoft/api-extractor": "^7.22.2",
+		"@octokit/core": "^4.0.5",
```

Comment:
It would be good to explain why this change is needed somewhere.

Reply:
We can't add comments in here. This is needed as per pnpm's warning of this being a missing peer dependency.